hummingbird/ml/_container.py

```diff
@@ -390,8 +390,18 @@ def _torchscript_wrapper(device, function, *inputs):
 
         # Maps data inputs to the expected type and device.
         for i in range(len(inputs)):
+            if type(inputs[i]) is list:
+                inputs[i] = np.array(inputs[i])
             if type(inputs[i]) is np.ndarray:
-                inputs[i] = torch.from_numpy(inputs[i]).float()
+                # Convert string arrays into int32.
+                if inputs[i].dtype.kind in {"U", "S"}:
```

Comment:
can you make these SUPPORTED_STRING_TYPES (also line 527 in this file, and _topology in two places as well?)

Reply:
good catch, I use vscode search but apparently it didn't find these